### ISSUE TITLE
substates do not record parent state info

### ISSUE THREAD

**Issue by doricon** [paleobot/pbot-dev#144]
When entering character instances, if you chose a substate, the parent state is not recorded. This info is very important to have recorded, can this be automated? 

For example: 
character: Margin type 
State untoothed
State toothed
     substate dentate
     substate serrate
     substate crenate. 

If I choose dentate, the fact that it is "toothed" is not recorded. Currently I would have to make two character instances to record the margin type being toothed:dentate. 

We need to have parent states recorded for search and matching purposes, given that for some fossils you may know the parent state but not the substate.

**Comment by aazaff** [comment]
Will this always be the case - i.e., will we ALWAYS want the state to substate relationship in ALL schemas?

**Comment by doricon** [comment]
Yes, Botany team has conferred, and yes definitely we will ALWAYS want the parent states recorded too.
